api/models/user_model.js

```diff
@@ -1,14 +1,44 @@
+
+
 class User{
-    constructor(id, email, firstName, lastName, hashedPassword, address, isAdmin){
-        this.id = id;
+    constructor(email, firstName, lastName, hashedPassword, address, isAdmin){
         this.email = email;
         this.firstName = firstName;
         this.lastName = lastName;
         this.password = hashedPassword;
         this.address = address;
         this.status = 'unverified';
-        this.isAdmin = isAdmin;
+        this.isAdmin = 'true';
     }
 }
 
-module.exports = User;
\ No newline at end of file
+module.exports = User;
+
+
+// app.post('/student', (req, res) => {
```

Comment:
Why is all these commented out?
You should delete if there is no need for them.